README.md

```diff
@@ -38,13 +40,90 @@ class User < ApplicationRecord
 end
 ```
 
-And then it will offer bunch of convenient methods and callbacks that ensure proper state transitions.
+It then will offer a bunch of convenient methods and callbacks that ensure proper state transitions.
+Note the prefix here to prefix the method. This is optional of course.
+
+```ruby
+user = User.new(state: 'active')
+# with the prefix: :state
+user.state_active?  # => true
+# or without the prefix: :state
+user.active? # => true
```

Comment:
Nit: these are features of the standard Rails `enum`

Reply:
Eeh yes, but only if you add an `enum` method in your model. (enum :state, yadiyada). With state_machine_enum this already gets added without the need to define an enum on a model as well. So it could be nice to know that we add it.

Reply:
Yes was planning on adding that. 👍